src/content/styleguide.mdx

```diff
@@ -42,40 +43,71 @@ contact-form-component.stories.tsx
 
-Keep the component related files in one catalogue for ease of finding the relation and working on a single module.
+The module has no obligatory structure. It may contain only one file or the whole app-like structure containing store, sub-components, internal routing, helpers etc. However, there are few rules that should be applied (if needed):
+- `<module>/components` is an obvious directory to store all components used in given module
+- `<module>/store` should contain all store related stuff
+- `<module>/img` is for storing images, SVGs, etc.
+- `<module>/constants` should contain constants used only in given module
+- `<module>/helpers` should contain all helper functions used across the module, including its store and sub-components; 
+  **it means there should be no `<module>.helpers.ts` files (or similar) as each helper function should be a separate file with its own `test` attached**
 
-### Examples
+> **Note:** Each module should be treated as an independent _package_ that can be easily transferred to other project by just copying files and using its API. This imply that all components, constants, store etc. used in given module should be encapsulated. However, this is the best-case scenario so don't try to follow this rule at all costs. Just switch your mindset to that approach and write the code that will be as modular as possible :wink:
 
-```typescript
-├── button
-│   ├── button.component.tsx
-│   ├── button-component.test.tsx
-│   ├── button-component.stories.tsx
-│   └── button-content.interface.ts
-```
+### Examples
 
-Modules should be placed in a flat structure of directories, which should be similar to how storybook is organised.
-[More about Storybook organisation](/stories#stories-structure)
-
-- `/server-functions` is a place to store a source code of all Netlify functions. Remember to keep this file flat and containing only endpoints, because of Netlify specification.
-- `/server-functions-tests` tests of stored above functions should be placed here. This is an exception from the base rule. Netlify doesn't allow to keep test files in functions folder.
-- `/hooks` - put all hooks here.
-- `/templates` - all templates for generated pages should be stored here.
-- `/pages-creators` - creators, which use above templates to create dynamic pages.
-- `/source-nodes-creators` - creators, which generate Gatsby source nodes
-- `/pages` - all static pages
-- `/images` - all static images if they are needed
-- `/constants` - all constants used in the app
+- a `calendar` module
+  ```
+   ── calendar
+      ├── calendar.container.tsx
+      ├── calendar.component.tsx
```

Comment:
Why it's not placed in `/components` if it's exactly a component?

Reply:
`components` is a directory for sub-components of given module, the main components of the module itself should be in its root directory

Reply:
@igorbezsmertnyi isn't it a `containers` case?

Reply:
We're not using sth like `container` anymore.

Reply:
@ulaulala What about this? I'm about to merge the changes

Reply:
IMO it would be nice to have a separate folder for main components instead of storing them in root, just for easier searching through the folders tree. 

Feel free to resolve if it's not a good idea for you and other devs